src/components/Card.tsx

```diff
@@ -88,15 +96,14 @@ const cardContainer = (
       'py-5': paddingType === 'normal',
       'py-8': typeof paddingType === 'undefined',
     }),
-    width(
-      thin ? 'sm:!w-thin-card' : 'sm:w-card',
-      thin ? 'tiny:w-thin-mobile' : 'w-mobile-card',
-      { 'w-32': thin }
-    ),
+    !useAppStyles ? (thin ? thinWidthStyles : defaultWidthStyles) : undefined,
     margin({ 'mx-auto': !thin }, 'lg:mx-0'),
-    maxHeight({ 'sm:max-h-card': !onlyWrap, 'max-h-mobile-card': !onlyWrap }),
+    maxHeight({
+      'sm:max-h-card': !onlyWrap && !useAppStyles,
```

Comment:
Seems like you do `!onlyWrap && !useAppStyles` twice, lets can extract this into `const`

Reply:
Fixed